courses/machine_learning/deepdive/08_image/mnist_linear.ipynb

```diff
@@ -9,9 +9,7 @@
    "source": [
```

Comment:
<ul><li>Use print(smth, smth, smth) instead of print(smth + " " + smth + " " + smth)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Ack.</p>